app/res/xml/preferences_developer.xml

```diff
@@ -85,6 +85,11 @@
         android:entryValues="@array/pref_enabled_vals"
         android:key="cc-auto-purge"
         android:title="Detect and Trigger Purge on Form Save"/>
+    <EditTextPreference
```

Comment:
That is a fair concern, but I don't think we want to get in the habit of implementing features sub-optimally just to make it easier to test them. I would much rather just add this to the manual test plan if it's really not feasible to test in calabash.